pkg/client/unversioned/urlbackoff_test.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package unversioned
+
+import (
+	"k8s.io/kubernetes/pkg/util"
+	"testing"
+	"time"
+	"net/url"
+)
+
+
+// parse is just a util for test readability.
+func parse(raw string) *url.URL{
+	theUrl , _ := url.Parse(raw)
+	return theUrl
+}
+
+func TestURLBackoffFunctionalityCollisions(t *testing.T) {
+	myBackoff := &URLBackoff{
+		Backoff: util.NewBackOff(1*time.Second, 60*time.Second),
+	}
+
+	//Add some noise and make sure backoff for a clean URL is zero...
```

Comment:
No, sorry for not being clear, I think you misunderstood me. I mean comments should start with a "//" and then have a " " before the text starts. There are numerous violations of this style rule in this PR. Please fix them-- thanks.